### PR TITLE
feat: add a reverse method

### PR DESCRIPTION
**Pull request by gustavoguichard**
I took over #71 to solve some conflicts and make some adjustments. I hope @mjuksel is ok with it =)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -538,6 +539,18 @@ const result = upperCase(str)
...
+### reverse
+
+This function reverses a string.
+
+```ts
+import { reverse } from 'string-ts'
+
+const str = 'Hello StringTS!'
+const result = reverse(str)
+//    ^ '!TSgnirtS olleH'
```
Comment (jly36963) [inline README.md:551]: STgnirtS (T and S are transposed)

**Review by jly36963 — APPROVED** [review]

**Comment by mjuksel** [comment]
> I took over #71 to solve some conflicts and make some adjustments. I hope @mjuksel is ok with it =)

👍 ofc men!

**Comment by jly36963** [comment]
Heck yeah, the Tsuchikage (@mjuksel) has spoken 🪨

**Comment by gustavoguichard** [comment]
@all-contributors please add @mjuksel for code, ideas.

**Pull request merged**